File: app/src/main/java/com/velaphi/untamed/features/animalList/models/Description.java
package com.velaphi.untamed.features.animalList.models;

import com.google.firebase.firestore.IgnoreExtraProperties;

@IgnoreExtraProperties
public class Description {
    private String heading;
    private String details;

    public Description(String heading, String details) {
        this.heading = heading;
        this.details = details;
    }

    public Description() {
    }

    public String getHeading() {
        return heading;
    }

    public void setHeading(String heading) {
        this.heading = heading;
    }

    public String getDetails() {
        return details;
    }

    public void setDetails(String details) {
        this.details = details;
    }
}

File: app/src/main/java/com/velaphi/untamed/features/all_animals/models/Description.kt
package com.velaphi.untamed.features.all_animals.models

import android.os.Parcelable
import com.google.firebase.firestore.IgnoreExtraProperties
import kotlinx.android.parcel.Parcelize

@IgnoreExtraProperties
@Parcelize
class Description(val content: String? = null,
                       val source: String? = null,
                       val link: String? = null,
                       val title: String? = null) : Parcelable